Sign-up.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jordan| Login-Signup</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="login.css">
+</head>
+
+<body>
+    <div class="header">
+        <div class="container">
+            <div class="navbar">
+                <div class="logo">
+                    <img src="images/logo.jpg" class="logo-Img">
+                </div>
+                <nav>
+                    <ul>
+                        <li><a href="Index.html">HOME</a></li>
+                        <li><a href=""></a>PRODUCTS</li>
+                        <li><a href=""></a>CART<span class="material-icons">
+                                shopping_cart
+                            </span></li>
+                        <li><a href=""></a>WISHLIST<span class="material-icons">
+                                favorite
+                            </span></li>
+                        <li><a href="login.HTML">LOGIN</a></li>
+                    </ul>
+                </nav>
+            </div>
+
+        </div>
+    </div>
+
+    <!--------------------------------------------Sign-UP------------------------------------------ -->
+    <div class="login-page">
+        <div class="container">
+            <div class="row">
+                <div class="col-2">
+                    <img src="IMAGES/home.jpg" class="Img-login">
+                </div>
+                <div class="col-2 col-form">
+                    <div class="form-container register-form">
+                        <div class="form-btn">
+                            <span class="Register-content">Not Registered Yet!</span>
+                        </div>
+                        <form>
+                            <input type="text" placeholder="UserName" max="10" min="4" required>
```

Comment:
use label tags

Reply:
did not used intentionally want the plain look